repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -121,6 +124,7 @@
                     <xs:element name="artifact-directory" type="papic:ArtifactDirectory" minOccurs="1" maxOccurs="1"/>
                     <xs:element name="ssl-configuration" type="papic:SslConfiguration" minOccurs="0" maxOccurs="1"/>
                     <xs:element name="logging-configuration" type="papic:LoggingConfiguration" minOccurs="0" maxOccurs="1"/>
+                    <xs:element name="via-configuration" type="papic:ViaConfiguration" minOccurs="0" maxOccurs="1"/>
```

Comment:
I don't really have an opinion here.

Now that Mario pointed out the inconsistencies, though, I cannot unsee them.